src/main/java/net/mojodna/osm2orc/standalone/parser/ChangesetElementProcessor.java

```diff
@@ -0,0 +1,120 @@
+package net.mojodna.osm2orc.standalone.parser;
+
+
+import net.mojodna.osm2orc.standalone.model.Changeset;
+import org.openstreetmap.osmosis.core.domain.common.TimestampContainer;
+import org.openstreetmap.osmosis.core.domain.v0_6.Tag;
+import org.openstreetmap.osmosis.xml.common.BaseElementProcessor;
+import org.openstreetmap.osmosis.xml.common.ElementProcessor;
+import org.openstreetmap.osmosis.xml.v0_6.impl.TagElementProcessor;
+import org.openstreetmap.osmosis.xml.v0_6.impl.TagListener;
+import org.xml.sax.Attributes;
+
+import java.util.List;
+
+public class ChangesetElementProcessor extends BaseElementProcessor implements TagListener {
```

Comment:
¯\_(ツ)_/¯

I remember having my mind blown when I was first working with Android--there was an abstraction over the SAX interface (I think; it was definitely XML parsing) that allowed one to cleanly implement it using anonymous interface implementations. If [something like] that has progressed even more, it might be a clean, easy replacement.

But hey, this works.